### PR TITLE
Add desktop file, custom launch script and an installer for them

### PR DESCRIPTION
**Pull request by mfdorst**
In order to launch umberwm from a display manager (such as gdm) you need a `.desktop` file in `/usr/share/xsessions`. This provides that file.

It also provides a launcher script `umberwm-start` which allows the user to run custom scripts on launch (personally I have to do a few `xrandr` and `amixer` commands in order to use the correct display and sound outputs. Currently the script just calls `umberwm`, but it being there makes the launch behavior easy to customize.

It also provides an installer script which will do three things:
1. Symlink the `umberwm` binary to `/usr/bin`
2. Symlink `umberwm-start` to `/usr/bin`
3. Copy `umberwm.desktop` to `/usr/share/xsessions` (it cannot be a symlink)

### CODE REVIEW

**Comment by mfdorst** [comment]
Oh I made a dumb mistake. I moved `main.rs` into this repo, since I don't see any point in it being separate. My installer script therefore assumes that the binary will be in `umberwm/target/release`, but obviously that is not the case. Oops.

**Comment by mfdorst** [comment]
Since it looks like we are adding `main.rs` to this repo as per #19, this can be reopened.

**Comment by mfdorst** [comment]
I suppose `umberwm-start` should be modified to reflect #25.

**Comment by yazgoo** [comment]
Agreed.
Also, could you please mention install script in README.md ?

**Comment by mfdorst** [comment]
Sure thing

**Comment by mfdorst** [comment]
I added the auto-restart code to `umberwm-start`, so that is preconfigured to work if you use the install script.

File: README.md [hunk 1]
```diff
@@ -35,25 +35,19 @@ Ubuntu: `sudo apt install libxcb-randr0-dev`.
...
-        [Desktop Entry]
-        Encoding=UTF-8
-        Name=UmberWM
-        Comment=Rusty window manager
-        Exec=/path/to/umberwm
-        Type=XSession
-        ```
+4. Edit `umberwm-start` if desired. Here you can launch any programs you need to before launching
+   `umberwm`.
+5. Run `cargo build --release`. The binary will be available in `target/release/umerwm`.
```
Comment (yazgoo) [inline README.md]:
```suggestion
5. Run `cargo build --release`. The binary will be available in `target/release/umberwm`.
```

File: README.md [hunk 2]
```diff
@@ -35,25 +35,19 @@ Ubuntu: `sudo apt install libxcb-randr0-dev`.
...
+6. Run `./install.py`. This will do three things:
```
Comment (yazgoo) [inline README.md]:
```suggestion
6. Optionally, run `./install.py`. This will do three things:
```

**Review by yazgoo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mfdorst) [reply README.md]: Oops lol

File: README.md (hunk 2 above)
Comment (mfdorst) [reply README.md]: Good change, will do

**Pull request merged**